docs/Developers/Executing_Tests.rst

```diff
@@ -56,8 +56,12 @@ To run our tests, you can also use ``python3 setup.py test``
 
 .. note::
 
-    You will not get a test coverage of 100% - the coverage on the
-    website is merged for several python versions.
+    You will not get a test coverage of 100% locally! - the coverage is
```

Comment:
Are you certain that the coverage will never be 100% locally? otherwise it should be 'You may not get a test coverage of 100% locally.'?